src/js/visual/configurator-csr.js

```diff
@@ -58,6 +58,7 @@ ripe.ConfiguratorCsr.prototype.init = function() {
     ripe.Visual.prototype.init.call(this);
 
     // options variables
+    this.id = this.options.id ?? 0;

```

Comment:
I think we should maintain the behavior consistent, but cool trick 😄

I would also suggest to have maintain this javascripty and let the ID be a string, gives more flexability